jvm/controls/src/main/resources/css/add-plugin-dialog.css

```diff
@@ -46,16 +45,16 @@
 }
 
 .add-plugin-dialog__btn--close:hover {
-    -fx-background-color: -wa-gray-lighter;
+    -fx-background-color: -wa-foreground-hover;
 }
 
 .add-plugin-dialog__btn--close .ikonli-font-icon {
     -fx-icon-size: 24px;
-    -fx-fill: -wa-gray;
+    -fx-fill: -wa-border-neutral;
```

Comment:
Not a good name for fill rule, it's not a border. I think it's ok if there will be multiple variables with the same values

Reply:
yeah I think -gray-neutral sounds better than -border

Reply:
yep, I added another css variable for that